Cargo.toml

```diff
@@ -59,3 +59,6 @@ codegen-units = 1
 incremental = true
 lto = true
 opt-level = "z"
+
+[patch.crates-io]
+deno_ast = { path = "../deno_ast" }
```

Comment:
This was probably accidentally committed.

Reply:
It's just temporary until the deno_ast changes land: https://github.com/denoland/deno_ast/pull/230